data/2015/other/Academic Policies & Procedures.xhtml

```diff
@@ -241,27 +234,19 @@
 </tbody>
 </table>
 <p class="title-3">Dismissal</p>
+<p>Status AfterAttempted UNL Credit HoursPrevious Semester NU Cumulative GPAUNL Term GPANU Cumulative GPAProbation &amp; Dismissal</p>
```

Comment:
This is the table headers row. It should not have been changed into a paragraph.